### ISSUE TITLE
Publish on Hackage

### ISSUE THREAD

**Issue by bergus** [ajnsit/FactorCSS#1]
Hi,
I've searched around the web and so far this module seems to be the only one being capable of parsing import declarations.
I would like to use it in my project, but cannot see how to import it or properly declare it as a dependency (apart from simply copying over the files, which I want to avoid).
Do you plan to publish it as an official module?

**Comment by ajnsit** [comment]
Good point. I've also wanted to put this lib up on Hackage, but never got around to it. I'll talk to the original author of the lib for permission and will probably release it in a few days.

**Comment by bergus** [comment]
To be fair, I found [wereHamster/haskell-css-syntax](//github.com/wereHamster/haskell-css-syntax) in the meantime, and while it's <s>parser</s> tokenizer is not as sophisticated as yours, it did suffice for my project. So if you still have not much time, please don't feel pushed by me.
